src/core/helpers/utils.ts

```diff
@@ -10,3 +10,15 @@ export const formatTime = (date: Date) => {
   const options: Intl.DateTimeFormatOptions = { hour: '2-digit', minute: '2-digit' };
   return date.toLocaleTimeString('fr-FR', options);
 };
+
+export const displayMinutes = (date: Date) => {
+  if (date.getMinutes() < 10) return '0'.concat(date.getMinutes().toString());
+  return date.getMinutes();
+};
+
+export const displayCivility = (civility: string) => {
```

Comment:
plutot que de faire une methode du peut mettre dans les constantes :  
```
  MISTER: 'mr',
  MRS: 'mrs',
  COUPLE: 'couple',
  CIVILITY_LIST() {
    return {
      [this.MISTER]: 'M.',
      [this.MRS]: 'Mme',
      [this.COUPLE]: 'Mme et M.',
    };
  },
```

et du coup dans le  template mettre directement `CIVILITY_OPTIONS[civility]`

Reply:
Je l'ai écrit un peu différemment en créant un tableau d'objet contenant les différentes civilités, qu'en penses-tu?

Reply:
ce format vient a l'origine des selecteurs et donc assez propre a cette utilisation 
en plus ca conplexifie beaucoup l'acces a la donnée `CIVILITY_OPTIONS.find(opt => opt.value === civility)?.label`
dans l'autre cas tu fais simplement `CIVILITY_OPTIONS[civiilty]` ce qui est plus lisible je trouve

Reply:
a pardon, j'ai mal ecrit ma proposition
```
export const MISTER = 'mr';
export const MRS = 'mrs',
export const COUPLE ='couple',
export const CIVILITY_OPTIONS = {
  [this.MISTER]: 'M.',
  [this.MRS]: 'Mme',
  [this.COUPLE]: 'Mme et M.',
};
```

Reply:
J'avais essaye tout a l'heure de l'écrire comme ca mais j'ai une erreur typescript que je n'arrive pas a résoudre: '`this` is possibly undefined '